spk-config.yaml

```diff
@@ -3,9 +3,9 @@ azure_devops:
   access_token: "hpe3a9oiswgcodtfdpzfiek3saxbrh5if1fp673xihgc5ap467a" # This is your Personal Access Token with permission to modify and access this private repo. Leave this empty if project is public
   hld_repository: "dev.azure.com/bhnook/fabrikam/_git/hld" # Repository URL for your Bedrock HLDs
   manifest_repository: "dev.azure.com/bhnook/fabrikam/_git/materialized" # Repository URL that is configured for flux. This holds the kubernetes manifests that is generated by fabrikate.
-  org: "bhnook" # Your AzDo Org
+  org: "epicstuff" # Your AzDo Org
   project: "fabrikam" # Your AzDo project
-
+  variable_group:
```

Comment:
should there be a sample variable group here?

Reply:
we can add some random name similar to other variables.